### PR TITLE
Help video changes

### PR DESCRIPTION
**Pull request by na9da**
### What this PR does

* Add `videoCoverImageOpacity` option to `HelpContentItem` so that we can fade the background of help video panels.
* Fix a bug where all `HelpVideoPanel`s were being rendered simultaneously resulting in autoplayed videos playing at random.

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated relevant documentation in `doc/`.
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've provided instructions in the PR description on how to test this PR.

### CODE REVIEW

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx [hunk 1]
```diff
@@ -49,15 +50,15 @@ class HelpVideoPanel extends React.Component {
       [Styles.shiftedToRight]:
         !isExpanded ||
         !this.props.viewState.showHelpMenu ||
-        this.props.viewState.topElement !== "HelpPanel",
-      [Styles.isHidden]: !itemSelected // when the item isn't selected
+        this.props.viewState.topElement !== "HelpPanel"
     });
     return (
-      <div className={className}>
+      itemSelected && <div className={className}>
```
Comment (na9da) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Render this component only when it is selected. Not really sure why it was implemented this way. In my testing with videos that have autoplay enabled, this resulted in the user clicking one video but another video gets played.

File: lib/ReactViewModels/defaultHelpContent.ts [hunk 2]
```diff
@@ -22,6 +22,9 @@ export interface HelpContentItem {
   videoUrl?: string;
   placeholderImage?: string;
 
+  // The `placeholderImage` is also used as background cover image for the container that embeds the video. This setting allows us to control the opacity of the cover image.
+  videoCoverImageOpacity?: boolean;
```
Comment (tephenavies) [inline lib/ReactViewModels/defaultHelpContent.ts]: Should be `number`

**Review by tephenavies — DISMISSED** [review]

File: lib/ReactViewModels/defaultHelpContent.ts (hunk 2 above)
Comment (na9da) [reply lib/ReactViewModels/defaultHelpContent.ts]: Ah yes!

**Comment by mwu2018** [comment]
I've verified it with `dev` site.

**Review by mwu2018 — APPROVED** [review]

**Pull request merged**